integration_test/util/auth.js

```diff
@@ -0,0 +1,35 @@
+const puppeteer = require('puppeteer')
+
+async function authenticateKeycloak (username, password) {
+  const browser = await puppeteer.launch()
+  const page = await browser.newPage({args: ['--no-sandbox', '--disable-setuid-sandbox']})
+
+  async function loginProcess () {
```

Comment:
When the password is wrong (imagine somebody changed the password on security.feedhenry.org), this method runs indefinitely. Maybe it is better to just fail if the login was not successful.

I was testing the password here to see what happens: https://github.com/aerogear/data-sync-server/pull/88/files#diff-6719cfa6ff7e6b161fc5f40bfe34efb7R54